lib/ui/myplot/new_my_plot_child_item.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'my_plot_page.dart';
+
+class MyNewPlotListItem {
+  Widget buildListItem(CropEntity cropsData, HomeMyPlotPageStyle myPlotStyle) {
+
+    //FIXME: This variables should be with the cropData, but for the moment are hardcored coz of the CMS Data.
+    const DayText = "Day 6";
+    const currentStage = "Planting";
+
+    return Column(
+      children: <Widget>[
+        Container(
+          padding: myPlotStyle.listPadding,
+          child: Row(
+            mainAxisAlignment: myPlotStyle.mainAxisAlignmentSpaceBeetwen,
+            children: <Widget>[
+              Column(
+                crossAxisAlignment: myPlotStyle.crossAxisAlignmentStart,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Text(
+                          cropsData.name ?? Strings.defaultCropNameText,
+                          style: myPlotStyle.cropNameTextStyle),
+                      Padding(
+                        padding: myPlotStyle.paddingBetweenElements,
+                      )
+                    ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Text(currentStage, style: myPlotStyle.cropStatusTextStyle),
+                    Padding(
+                      padding: myPlotStyle.paddingBeforeCropDayCount,
+                    )
+                  ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Container(
+                      padding: myPlotStyle.paddingForDayCount,
+                      decoration: BoxDecoration(
+                        color: myPlotStyle.primaryGreen.withOpacity(myPlotStyle.oppacityForDayCount),
+                        borderRadius: myPlotStyle.ovalRadiousForDayCount),
+                      child: Row(
+                        mainAxisSize: myPlotStyle.mainAxisSizeMin,
+                       children: <Widget>[
+                         Flexible(
+                           child: Container(
+                             child: Text(
+                              DayText,
+                              style: myPlotStyle.cropDayTextStyle,
+                           ),
+                         )
+                         )],
+                      ),
+                    )
+                  ],
+                  ),
+                ],
+              ),
+              Column(
+                children: <Widget>[
+                  _buildClipOval(cropsData, myPlotStyle),
+                ],
+              )
+            ],
+          ),
+        ),
+        Container(
+            child: buildListSeparator(myPlotStyle)
+        ),
+      ],
+    );
+  }
+
+  ClipOval _buildClipOval(CropEntity cropsData, HomeMyPlotPageStyle myPlotStyle) {
+    return ClipOval(
+                  child: NetworkImageFromFuture(
```

Comment:
this is for another PR, but I think we should have our own ImageWidget so we can add in hints for image size requests. (e.g if the widget is small it it will request a small image, if big it will request a bigger one) But this is for another task.